IndoorMapsBackend/src/utils/websocketServer.ts

```diff
@@ -169,7 +169,7 @@ export const server = net.createServer(sock => {
         magicString = crypto.createHash('sha1').update(magicString).digest('base64');
 
         const wsKey = crypto.randomUUID()
-        const jwtCookie = cookiesHeader.split(";").find(cookie => cookie.startsWith("jwt="));
+        const jwtCookie = cookiesHeader.split("; ").find(cookie => cookie.startsWith("jwt="));
```

Comment:
[Stretch] same here, as long as it is searching within a string this can be made easier by using regex